backend/server/static/api/meetingbook.yaml

```diff
@@ -163,22 +164,230 @@ paths:
         "401":
           description: "Unauthorized"
       security:
-        - basic_auth: []    
-  /bookings/{booking_id}:
+        - basic_auth: []
+  /booking_settings:
+    summary: endpoint for the settings of the booking, controlled by admin.
```

Comment:
/blabla should NOT have additional properties like summary

Reply:
> /blabla should not have additional properties like summary

for block description can use structure like:
tags:
- name: booking_settings
  description: endpoint for the settings of the booking, controlled by admin.
and add in get, put, post:
tags:
 -booking_settings

Reply:
https://swagger.io/docs/specification/grouping-operations-with-tags/

Reply:
Ack